dist/main.js

```diff
@@ -13,22 +13,21 @@ function onButtonClick() {
 		saveButton.innerHTML = 'Item Saved!';
 	}
 
-	messageInput.innerHTML = '<div>' + messageInput.value + '</div>';	
+	messageInput.innerHTML = '<div>' + messageInput.value + '</div>';
 
 }
 
 ///////////////////////////////////////////////////
 
 
-var input = document.getElementById("todoInput");
```

Comment:
This variable was a reference to the same thing as `messageInput` on line 2, so I just removed this one an renamed all instance of `input` to `messageInput` (lines 25, 29, and 30).